### PR TITLE
TMA-1208 Log to sigle file from K8s LCM bricks

### PR DESCRIPTION
**Pull request by sadam21**
Logs will be written to NFS and visible in admin console.

### CODE REVIEW

File: lib/gooddata/lcm/lcm2.rb [hunk 1]
```diff
@@ -372,7 +372,7 @@ def perform(mode, params = {})
 
       ensure
         if logging_enabled
-          brick_logger.log_action('finished', JSON.pretty_generate(result))
+          brick_logger.log_action(errors.empty? ? 'finished' : 'error')
```
Comment (yenkins) [inline lib/gooddata/lcm/lcm2.rb]: info: Calls 'errors.empty?' 2 times (DuplicateMethodCall) (reek)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/870/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1683/ : SUCCESS in 2m 39s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1944/ : SUCCESS in 3m 00s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/121/ : SUCCESS in 7m 35s

File: lib/gooddata/lcm/lcm2.rb [hunk 2]
```diff
@@ -372,7 +370,8 @@ def perform(mode, params = {})
 
       ensure
         if logging_enabled
-          brick_logger.log_action('finished', JSON.pretty_generate(result))
+          brick_logger.log_action(JSON.pretty_generate(errors)) if errors.any?
```
Comment (yenkins) [inline lib/gooddata/lcm/lcm2.rb]: info: Calls 'json.pretty_generate(errors)' 2 times (DuplicateMethodCall) (reek)

Comment (yenkins) [inline lib/gooddata/lcm/lcm2.rb]: info: Calls 'errors.any?' 2 times (DuplicateMethodCall) (reek)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/878/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1691/ : SUCCESS in 3m 06s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1952/ : SUCCESS in 2m 45s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/129/ : SUCCESS in 8m 04s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/879/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1692/ : SUCCESS in 1m 21s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1953/ : SUCCESS in 1m 20s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/130/ : SUCCESS in 7m 25s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/881/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1694/ : SUCCESS in 2m 39s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1955/ : SUCCESS in 2m 58s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/132/ : SUCCESS in 7m 53s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/147/ : SUCCESS in 19s

**Pull request merged**